shared/src/version/mod.rs

```diff
@@ -50,6 +51,10 @@ impl NetworkVersion {
     pub const V17: Self = Self(17);
     /// Hygge (builtin-actors v10)
     pub const V18: Self = Self(18);
+    /// Lightning (builtin-actors v11)
+    pub const V19: Self = Self(19);
+    /// Lightning (builtin-actors v11)
```

Comment:
Idk how we're thinking about actors versioning now but wouldn't this be v12?

Reply:
No, because FIP-0061 proposes a 2-stage upgrade, with a rollover period.

Reply:
Call the second one Thunder, maybe?

Reply:
That's such a good idea, it's worth re-running CI.